app/views/content/life-as-a-teacher/teaching-as-a-career/abigails-career-progression-story/_article.html.erb

```diff
@@ -0,0 +1,24 @@
+      <p>I really enjoyed my teacher training. Although I had no experience in the classroom beforehand, I found I settled quite naturally into speaking to pupils at the front of the classroom. I also had a lot of support from my mentors and the maths team.</p>
+      <h2>Taking on a new role</h2>
+      <p>In the January of my early career teacher (ECT) year, my school advertised a numeracy coordinator role. The role was all about embedding numeracy into subjects across the whole school. Despite being an ECT, I was keen to progress and had a lot of ideas for the role, so I decided to go for it. I worked hard to prepare and I was so pleased to be offered the job!</p>
+      <p>The numeracy coordinator role was a new one in the school, so I could really make it my own. I decided to cement numeracy into form time, coming up with activities pupils could do to improve basic skills, and I worked with teachers in different departments to make sure we were being consistent in the way we were teaching maths. I also introduced a maths mastery homework competition across key stage 3. I had a lot of support and encouragement from my mentor and from teaching staff across the school.</p>
+      <h2>Increased responsibility</h2>
+      <p>In my third year I was promoted to second in department and took on more responsibility. I lead meetings, manage the budget, conduct appraisals, and organise ‘learning walks’ to ensure teaching is consistent across the department. This extra work is reflected in my teaching schedule, with reduced classroom hours so that I can manage all the additional tasks.</p>
+      <p>What I like most about being in a leadership role is having more responsibility for key decisions, improving pedagogy and empowering others.</p>
+  </section>
+  <section class="col col-720">
+    <%= render Content::QuoteComponent.new(
+    text: "My influence and decisions are ultimately impacting on our pupils’ lives and the experience that they have at school.",
+    large: true
+  ) %>
+  </section>
+  <section class="col col-720 col-space-l-top">
+      <p>The main goal is to help pupils leave school with the best possible GCSE results. Yet teachers are also responsible for producing confident, caring, respectful and well-rounded individuals ready to take the next steps in their educational journeys.</p>
+      <p>This year I’ve been doing the National Professional Qualification for Senior Leadership (NPQSL), which has involved running a project across the whole school. This has given me the opportunity to work more closely with the senior leadership team and has helped strengthen my relationships with colleagues in other departments in the school.</p>
```

Comment:
in the welcome guide article about Abigail, it says she already has this - can we merge that info in?

/welcome/my-journey-into-teaching

(need to keep the welcome guide as is for now)